inject.js

```diff
@@ -393,11 +378,17 @@
   function runAction(action, document, value, e) {
     var videoTags = document.getElementsByTagName('video');
     videoTags.forEach = Array.prototype.forEach;
-
+	if (e){
```

Comment:
Can you elaborate on what this logic is doing?

Reply:
When the controller buttons are pressed it passes the event e to runAction. If runAction is called by a keyBinding than e is null and it will skip the following tests and change all video speeds.

We use the event to find the videoTag owning the controller. Rather than changing the speed of all videos we check each one to see if it has the same controller and only proceed if it does.

Reply:
Can you please add a comment to capture this in the source as well?